larq_compute_engine/mlir/python/converter.py

```diff
@@ -50,6 +52,7 @@ def _contains_training_quant_op(graph_def):
 def convert_keras_model(
     model: tf.keras.Model,
     *,  # Require remaining arguments to be keyword-only.
+    default_ranges: Optional[Tuple[float, float]] = None,
```

Comment:
Internally we ran into https://github.com/tensorflow/tensorflow/issues/40055

@AdamHillier how do you feel about marking `default_ranges` as experimental for now as well?

Reply:
Yes, I think we should definitely do that. `experimental_default_int8_range`?

Reply:
Sounds good. Changed it to `experimental_default_int8_range` in 483781ac4fd2eb194360c7b39ed40747c92ee08b